### PR TITLE
Added scripts for bft node and send transaction

### PR DESCRIPTION
**Pull request by dkijania**

### CODE REVIEW

File: scripts/bft-single-node-test [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+FAUCET_AMOUNT=1000000000
+ADDRTYPE="--testing"
+REST_PORT="8443"
+
+### MAKE EVERYTHING
+
+# faucet
+FAUCET_SK=$($CLI key generate --type=Ed25519Extended)
+FAUCET_PK=$(echo ${FAUCET_SK} | $CLI key to-public)
+FAUCET_ADDR=$($CLI address single ${ADDRTYPE} ${FAUCET_PK})
```
Comment (NicolasDP) [inline scripts/bft-single-node-test]: make the faucet an account

File: scripts/bft-single-node-test [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+echo  "--------"
+echo "To start the node as leader :"
+echo "$NODE --genesis-block block-0.bin --config config.yaml --secret secret.yaml"
+echo  "--------"
+echo "To start the node as standard node (without ability to create blocks) :"
+echo "$NODE --genesis-block block-0.bin --config config.yaml --secret secret.yaml"
+echo  "--------"
+echo "To interact with node :"
+echo "$CLI rest v0 node stats -h http://127.0.0.1:${REST_PORT}/api"
+echo  "--------"
```
Comment (NicolasDP) [inline scripts/bft-single-node-test]: there are coloring options available from the other script, reuse

File: scripts/bft-single-node-test (hunk 1 above)
Comment (dkijania) [reply scripts/bft-single-node-test]: AFAIK accounts are not working currently (cannot get account state ) . See https://github.com/input-output-hk/jormungandr/issues/388

File: scripts/bft-single-node-test (hunk 2 above)
Comment (dkijania) [reply scripts/bft-single-node-test]: There is this issue: https://github.com/input-output-hk/jormungandr/issues/441. Observed on my machine also. that's why I've removed colouring

Comment (vincenthz) [reply scripts/bft-single-node-test]: it has been fixed already

File: scripts/bft-single-node-test (hunk 1 above)
Comment (vincenthz) [reply scripts/bft-single-node-test]: they are working now

**Pull request merged**